libs/components/forms/testing/src/checkbox/checkbox-group-harness.spec.ts

```diff
@@ -239,9 +239,9 @@ describe('Checkbox group harness', () => {
 
     await checkboxHarness.check();
 
-    await expectAsync(
-      checkboxGroupHarness.hasError('test'),
-    ).toBeRejectedWithError('No form errors found.');
+    await expectAsync(checkboxGroupHarness.hasError('test')).toBeResolvedTo(
```

Comment:
I like this implementation better (resolving `false`), but this is a breaking change. In the harness, you could throw the same "No form errors found" error when child `sky-form-error` components cannot be found.

Reply:
Added a non-breaking version that we will want to take out in 11